diff -r -c target/ya_market/api/provider_api.py src/ya_market/api/provider_api.py
*** target/ya_market/api/provider_api.py	2020-12-05 15:48:50.293000000 +0100
--- src/ya_market/api/provider_api.py	2020-12-05 15:51:50.763920739 +0100
***************
*** 102,108 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 102,108 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 236,242 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 236,242 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 366,372 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 366,372 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 496,502 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 496,502 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 1124,1130 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 1124,1130 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 1385,1391 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 1385,1391 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 1511,1517 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 1511,1517 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 1634,1640 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 1634,1640 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
Only in src/ya_market/api: __pycache__
diff -r -c target/ya_market/api/requestor_api.py src/ya_market/api/requestor_api.py
*** target/ya_market/api/requestor_api.py	2020-12-05 15:48:50.293000000 +0100
--- src/ya_market/api/requestor_api.py	2020-12-05 15:51:50.743920898 +0100
***************
*** 217,223 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 217,223 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 346,352 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 346,352 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 476,482 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 476,482 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 605,611 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 605,611 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 729,735 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 729,735 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 1623,1629 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 1623,1629 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 1749,1755 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 1749,1755 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
***************
*** 1872,1878 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 1872,1878 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
diff -r -c target/ya_market/models/agreement_approved_event.py src/ya_market/models/agreement_approved_event.py
*** target/ya_market/models/agreement_approved_event.py	2020-12-05 15:48:50.257000000 +0100
--- src/ya_market/models/agreement_approved_event.py	2020-12-05 15:52:33.275588020 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_market.configuration import Configuration


--- 13,18 ----
***************
*** 49,55 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 47,53 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_market/models/agreement_cancelled_event.py src/ya_market/models/agreement_cancelled_event.py
*** target/ya_market/models/agreement_cancelled_event.py	2020-12-05 15:48:50.257000000 +0100
--- src/ya_market/models/agreement_cancelled_event.py	2020-12-05 15:52:33.479586429 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_market.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_market/models/agreement_operation_event.py src/ya_market/models/agreement_operation_event.py
*** target/ya_market/models/agreement_operation_event.py	2020-12-05 15:48:50.257000000 +0100
--- src/ya_market/models/agreement_operation_event.py	2020-12-05 15:52:33.503586241 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_market.configuration import Configuration


--- 13,18 ----
***************
*** 145,151 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 143,149 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_market/models/agreement_rejected_event_all_of.py src/ya_market/models/agreement_rejected_event_all_of.py
*** target/ya_market/models/agreement_rejected_event_all_of.py	2020-12-05 15:48:50.257000000 +0100
--- src/ya_market/models/agreement_rejected_event_all_of.py	2020-12-05 15:52:33.191588673 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_market.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_market/models/agreement_rejected_event.py src/ya_market/models/agreement_rejected_event.py
*** target/ya_market/models/agreement_rejected_event.py	2020-12-05 15:48:50.257000000 +0100
--- src/ya_market/models/agreement_rejected_event.py	2020-12-05 15:52:33.439586740 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_market.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_market/models/agreement_terminated_event_all_of.py src/ya_market/models/agreement_terminated_event_all_of.py
*** target/ya_market/models/agreement_terminated_event_all_of.py	2020-12-05 15:48:50.257000000 +0100
--- src/ya_market/models/agreement_terminated_event_all_of.py	2020-12-05 15:52:33.379587208 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_market.configuration import Configuration


--- 13,18 ----
***************
*** 137,143 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 135,141 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_market/models/agreement_terminated_event.py src/ya_market/models/agreement_terminated_event.py
*** target/ya_market/models/agreement_terminated_event.py	2020-12-05 15:48:50.257000000 +0100
--- src/ya_market/models/agreement_terminated_event.py	2020-12-05 15:52:33.451586648 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_market.configuration import Configuration


--- 13,18 ----
***************
*** 137,143 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 135,141 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_market/models/proposal_rejected_event_all_of.py src/ya_market/models/proposal_rejected_event_all_of.py
*** target/ya_market/models/proposal_rejected_event_all_of.py	2020-12-05 15:48:50.257000000 +0100
--- src/ya_market/models/proposal_rejected_event_all_of.py	2020-12-05 15:52:33.119589234 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_market.configuration import Configuration


--- 13,18 ----
***************
*** 104,110 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 102,108 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_market/models/proposal_rejected_event.py src/ya_market/models/proposal_rejected_event.py
*** target/ya_market/models/proposal_rejected_event.py	2020-12-05 15:48:50.257000000 +0100
--- src/ya_market/models/proposal_rejected_event.py	2020-12-05 15:52:33.295587864 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_market.configuration import Configuration


--- 13,18 ----
***************
*** 104,110 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 102,108 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
Only in src/ya_market/models: __pycache__
diff -r -c target/ya_market/models/reason.py src/ya_market/models/reason.py
*** target/ya_market/models/reason.py	2020-12-05 15:48:50.257000000 +0100
--- src/ya_market/models/reason.py	2020-12-05 15:52:33.359587364 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_market.configuration import Configuration


--- 13,18 ----
***************
*** 78,84 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 76,82 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
Only in src/ya_market: __pycache__
Only in src/ya_market/test: __pycache__
diff -r -c target/ya_market/test/test_agreement_event_all_of.py src/ya_market/test/test_agreement_event_all_of.py
*** target/ya_market/test/test_agreement_event_all_of.py	2020-12-05 15:48:50.289000000 +0100
--- src/ya_market/test/test_agreement_event_all_of.py	2020-12-05 16:05:43.504859222 +0100
***************
*** 34,67 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_market.models.agreement_event_all_of.AgreementEventAllOf()  # noqa: E501
          if include_optional :
              return AgreementEventAllOf(
                  agreement = ya_market.models.agreement.Agreement(
!                     agreement_id = '0',
!                     demand = null,
!                     offer = null,
!                     valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     approved_date = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     state = 'Proposal',
!                     timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     app_session_id = '0',
!                     proposed_signature = '0',
!                     approved_signature = '0',
                      committed_signature = '0', )
              )
          else :
              return AgreementEventAllOf(
                  agreement = ya_market.models.agreement.Agreement(
!                     agreement_id = '0',
!                     demand = null,
!                     offer = null,
!                     valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     approved_date = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     state = 'Proposal',
!                     timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     app_session_id = '0',
!                     proposed_signature = '0',
!                     approved_signature = '0',
                      committed_signature = '0', ),
          )

--- 34,69 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_market.models.agreement_event_all_of.AgreementEventAllOf()  # noqa: E501
+         demand = ya_market.models.DemandOfferBase(properties=dict(), constraints=str())
+         offer = ya_market.models.DemandOfferBase(properties=dict(), constraints=str())
          if include_optional :
              return AgreementEventAllOf(
                  agreement = ya_market.models.agreement.Agreement(
!                     agreement_id = '0',
!                     demand = demand,
!                     offer = offer,
!                     valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     approved_date = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     state = 'Proposal',
!                     timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     app_session_id = '0',
!                     proposed_signature = '0',
!                     approved_signature = '0',
                      committed_signature = '0', )
              )
          else :
              return AgreementEventAllOf(
                  agreement = ya_market.models.agreement.Agreement(
!                     agreement_id = '0',
!                     demand = demand,
!                     offer = offer,
!                     valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     approved_date = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     state = 'Proposal',
!                     timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     app_session_id = '0',
!                     proposed_signature = '0',
!                     approved_signature = '0',
                      committed_signature = '0', ),
          )

diff -r -c target/ya_market/test/test_agreement_event.py src/ya_market/test/test_agreement_event.py
*** target/ya_market/test/test_agreement_event.py	2020-12-05 15:48:50.289000000 +0100
--- src/ya_market/test/test_agreement_event.py	2020-12-05 16:07:12.759811126 +0100
***************
*** 34,67 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_market.models.agreement_event.AgreementEvent()  # noqa: E501
          if include_optional :
              return AgreementEvent(
                  agreement = ya_market.models.agreement.Agreement(
!                     agreement_id = '0',
!                     demand = null,
!                     offer = null,
!                     valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     approved_date = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     state = 'Proposal',
!                     timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     app_session_id = '0',
!                     proposed_signature = '0',
!                     approved_signature = '0',
                      committed_signature = '0', )
              )
          else :
              return AgreementEvent(
                  agreement = ya_market.models.agreement.Agreement(
!                     agreement_id = '0',
!                     demand = null,
!                     offer = null,
!                     valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     approved_date = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     state = 'Proposal',
!                     timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     app_session_id = '0',
!                     proposed_signature = '0',
!                     approved_signature = '0',
                      committed_signature = '0', ),
          )

--- 34,69 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_market.models.agreement_event.AgreementEvent()  # noqa: E501
+         demand = ya_market.models.DemandOfferBase(properties=dict(), constraints=str())
+         offer = ya_market.models.DemandOfferBase(properties=dict(), constraints=str())
          if include_optional :
              return AgreementEvent(
                  agreement = ya_market.models.agreement.Agreement(
!                     agreement_id = '0',
!                     demand = demand,
!                     offer = offer,
!                     valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     approved_date = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     state = 'Proposal',
!                     timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     app_session_id = '0',
!                     proposed_signature = '0',
!                     approved_signature = '0',
                      committed_signature = '0', )
              )
          else :
              return AgreementEvent(
                  agreement = ya_market.models.agreement.Agreement(
!                     agreement_id = '0',
!                     demand = demand,
!                     offer = offer,
!                     valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     approved_date = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     state = 'Proposal',
!                     timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                     app_session_id = '0',
!                     proposed_signature = '0',
!                     approved_signature = '0',
                      committed_signature = '0', ),
          )

diff -r -c target/ya_market/test/test_agreement.py src/ya_market/test/test_agreement.py
*** target/ya_market/test/test_agreement.py	2020-12-05 15:48:50.289000000 +0100
--- src/ya_market/test/test_agreement.py	2020-12-05 16:05:09.005278529 +0100
***************
*** 34,58 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_market.models.agreement.Agreement()  # noqa: E501
          if include_optional :
              return Agreement(
!                 agreement_id = '0',
!                 demand = null,
!                 offer = null,
!                 valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                 approved_date = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                 state = 'Proposal',
!                 timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                 app_session_id = '0',
!                 proposed_signature = '0',
!                 approved_signature = '0',
                  committed_signature = '0'
              )
          else :
              return Agreement(
                  agreement_id = '0',
!                 demand = null,
!                 offer = null,
                  valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
                  state = 'Proposal',
                  timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
--- 34,60 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_market.models.agreement.Agreement()  # noqa: E501
+         demand = ya_market.models.DemandOfferBase(properties=dict(), constraints=str())
+         offer = ya_market.models.DemandOfferBase(properties=dict(), constraints=str())
          if include_optional :
              return Agreement(
!                 agreement_id = '0',
!                 demand = demand,
!                 offer = offer,
!                 valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                 approved_date = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                 state = 'Proposal',
!                 timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
!                 app_session_id = '0',
!                 proposed_signature = '0',
!                 approved_signature = '0',
                  committed_signature = '0'
              )
          else :
              return Agreement(
                  agreement_id = '0',
!                 demand = demand,
!                 offer = offer,
                  valid_to = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
                  state = 'Proposal',
                  timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
diff -r -c target/ya_market/test/test_demand.py src/ya_market/test/test_demand.py
*** target/ya_market/test/test_demand.py	2020-12-05 15:48:50.285000000 +0100
--- src/ya_market/test/test_demand.py	2020-12-05 16:08:09.479169557 +0100
***************
*** 35,49 ****
          # model = ya_market.models.demand.Demand()  # noqa: E501
          if include_optional :
              return Demand(
!                 properties = None,
!                 constraints = '0',
!                 demand_id = '0',
!                 requestor_id = '0',
                  timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f')
              )
          else :
              return Demand(
!                 properties = None,
                  constraints = '0',
                  demand_id = '0',
                  requestor_id = '0',
--- 35,49 ----
          # model = ya_market.models.demand.Demand()  # noqa: E501
          if include_optional :
              return Demand(
!                 properties = dict(),
!                 constraints = '0',
!                 demand_id = '0',
!                 requestor_id = '0',
                  timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f')
              )
          else :
              return Demand(
!                 properties = dict(),
                  constraints = '0',
                  demand_id = '0',
                  requestor_id = '0',
diff -r -c target/ya_market/test/test_offer.py src/ya_market/test/test_offer.py
*** target/ya_market/test/test_offer.py	2020-12-05 15:48:50.285000000 +0100
--- src/ya_market/test/test_offer.py	2020-12-05 16:08:51.578704232 +0100
***************
*** 35,49 ****
          # model = ya_market.models.offer.Offer()  # noqa: E501
          if include_optional :
              return Offer(
!                 properties = None,
!                 constraints = '0',
!                 offer_id = '0',
!                 provider_id = '0',
                  timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f')
              )
          else :
              return Offer(
!                 properties = None,
                  constraints = '0',
                  offer_id = '0',
                  provider_id = '0',
--- 35,49 ----
          # model = ya_market.models.offer.Offer()  # noqa: E501
          if include_optional :
              return Offer(
!                 properties = dict(),
!                 constraints = '0',
!                 offer_id = '0',
!                 provider_id = '0',
                  timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f')
              )
          else :
              return Offer(
!                 properties = dict(),
                  constraints = '0',
                  offer_id = '0',
                  provider_id = '0',
diff -r -c target/ya_market/test/test_property_query_event_all_of.py src/ya_market/test/test_property_query_event_all_of.py
*** target/ya_market/test/test_property_query_event_all_of.py	2020-12-05 15:48:50.289000000 +0100
--- src/ya_market/test/test_property_query_event_all_of.py	2020-12-05 16:02:26.071387992 +0100
***************
*** 38,45 ****
          if include_optional :
              return PropertyQueryEventAllOf(
                  property_query = ya_market.models.property_query.PropertyQuery(
!                     issuer_properties = ya_market.models.issuer_properties.issuerProperties(),
!                     query_id = '0',
                      queried_properties = [
                          '0'
                          ], )
--- 38,45 ----
          if include_optional :
              return PropertyQueryEventAllOf(
                  property_query = ya_market.models.property_query.PropertyQuery(
!                     issuer_properties = dict(),
!                     query_id = '0',
                      queried_properties = [
                          '0'
                          ], )
***************
*** 47,54 ****
          else :
              return PropertyQueryEventAllOf(
                  property_query = ya_market.models.property_query.PropertyQuery(
!                     issuer_properties = ya_market.models.issuer_properties.issuerProperties(),
!                     query_id = '0',
                      queried_properties = [
                          '0'
                          ], ),
--- 47,54 ----
          else :
              return PropertyQueryEventAllOf(
                  property_query = ya_market.models.property_query.PropertyQuery(
!                     issuer_properties = dict(),
!                     query_id = '0',
                      queried_properties = [
                          '0'
                          ], ),
diff -r -c target/ya_market/test/test_property_query_event.py src/ya_market/test/test_property_query_event.py
*** target/ya_market/test/test_property_query_event.py	2020-12-05 15:48:50.289000000 +0100
--- src/ya_market/test/test_property_query_event.py	2020-12-05 16:05:55.496715420 +0100
***************
*** 36,43 ****
          if include_optional :
              return PropertyQueryEvent(
                  property_query = ya_market.models.property_query.PropertyQuery(
!                     issuer_properties = ya_market.models.issuer_properties.issuerProperties(),
!                     query_id = '0',
                      queried_properties = [
                          '0'
                          ], )
--- 36,43 ----
          if include_optional :
              return PropertyQueryEvent(
                  property_query = ya_market.models.property_query.PropertyQuery(
!                     issuer_properties = dict(),
!                     query_id = '0',
                      queried_properties = [
                          '0'
                          ], )
***************
*** 45,52 ****
          else :
              return PropertyQueryEvent(
                  property_query = ya_market.models.property_query.PropertyQuery(
!                     issuer_properties = ya_market.models.issuer_properties.issuerProperties(),
!                     query_id = '0',
                      queried_properties = [
                          '0'
                          ], ),
--- 45,52 ----
          else :
              return PropertyQueryEvent(
                  property_query = ya_market.models.property_query.PropertyQuery(
!                     issuer_properties = dict(),
!                     query_id = '0',
                      queried_properties = [
                          '0'
                          ], ),
diff -r -c target/ya_market/test/test_proposal_event_all_of.py src/ya_market/test/test_proposal_event_all_of.py
*** target/ya_market/test/test_proposal_event_all_of.py	2020-12-05 15:48:50.289000000 +0100
--- src/ya_market/test/test_proposal_event_all_of.py	2020-12-05 16:13:49.131621284 +0100
***************
*** 33,45 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_market.models.proposal_event_all_of.ProposalEventAllOf()  # noqa: E501
          if include_optional :
              return ProposalEventAllOf(
!                 proposal = null
              )
          else :
              return ProposalEventAllOf(
!                 proposal = null,
          )

      def testProposalEventAllOf(self):
--- 33,53 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_market.models.proposal_event_all_of.ProposalEventAllOf()  # noqa: E501
+         proposal = ya_market.models.proposal.Proposal(
+             properties=dict(),
+             constraints='0',
+             proposal_id='0',
+             issuer_id='0',
+             state='Initial',
+             timestamp='2020-12-01T08:00:00.00Z',
+         )
          if include_optional :
              return ProposalEventAllOf(
!                 proposal=proposal
              )
          else :
              return ProposalEventAllOf(
!                 proposal=proposal,
          )

      def testProposalEventAllOf(self):
diff -r -c target/ya_market/test/test_proposal_event.py src/ya_market/test/test_proposal_event.py
*** target/ya_market/test/test_proposal_event.py	2020-12-05 15:48:50.289000000 +0100
--- src/ya_market/test/test_proposal_event.py	2020-12-05 16:13:29.347817188 +0100
***************
*** 33,45 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_market.models.proposal_event.ProposalEvent()  # noqa: E501
          if include_optional :
              return ProposalEvent(
!                 proposal = null
              )
          else :
              return ProposalEvent(
!                 proposal = null,
          )

      def testProposalEvent(self):
--- 33,53 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_market.models.proposal_event.ProposalEvent()  # noqa: E501
+         proposal = ya_market.models.proposal.Proposal(
+             properties=dict(),
+             constraints='0',
+             proposal_id='0',
+             issuer_id='0',
+             state='Initial',
+             timestamp='2020-12-01T08:00:00.00Z',
+         )
          if include_optional :
              return ProposalEvent(
!                 proposal=proposal
              )
          else :
              return ProposalEvent(
!                 proposal=proposal,
          )

      def testProposalEvent(self):
diff -r -c target/ya_market/test/test_proposal.py src/ya_market/test/test_proposal.py
*** target/ya_market/test/test_proposal.py	2020-12-05 15:48:50.289000000 +0100
--- src/ya_market/test/test_proposal.py	2020-12-05 16:08:57.162643163 +0100
***************
*** 35,51 ****
          # model = ya_market.models.proposal.Proposal()  # noqa: E501
          if include_optional :
              return Proposal(
!                 properties = None,
!                 constraints = '0',
!                 proposal_id = '0',
!                 issuer_id = '0',
!                 state = 'Initial',
!                 timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
                  prev_proposal_id = '0'
              )
          else :
              return Proposal(
!                 properties = None,
                  constraints = '0',
                  proposal_id = '0',
                  issuer_id = '0',
--- 35,51 ----
          # model = ya_market.models.proposal.Proposal()  # noqa: E501
          if include_optional :
              return Proposal(
!                 properties = dict(),
!                 constraints = '0',
!                 proposal_id = '0',
!                 issuer_id = '0',
!                 state = 'Initial',
!                 timestamp = datetime.datetime.strptime('2013-10-20 19:20:30.00', '%Y-%m-%d %H:%M:%S.%f'),
                  prev_proposal_id = '0'
              )
          else :
              return Proposal(
!                 properties = dict(),
                  constraints = '0',
                  proposal_id = '0',
                  issuer_id = '0',
