diff -c a/ya_payment/api_client.py b/ya_payment/api_client.py
*** a/ya_payment/api_client.py	2020-06-29 00:39:33.397315375 +0200
--- b/ya_payment/api_client.py	2020-06-29 00:51:17.297582259 +0200
***************
*** 55,61 ****
      PRIMITIVE_TYPES = (float, bool, bytes, six.text_type) + six.integer_types
      NATIVE_TYPES_MAPPING = {
          'int': int,
!         'long': int if six.PY3 else long,  # noqa: F821
          'float': float,
          'str': str,
          'bool': bool,
--- 55,61 ----
      PRIMITIVE_TYPES = (float, bool, bytes, six.text_type) + six.integer_types
      NATIVE_TYPES_MAPPING = {
          'int': int,
!         'long': int, 
          'float': float,
          'str': str,
          'bool': bool,

