Poe => ./build-scripts/diff-module.sh market 003
diff -r -c ya-py-aioclient-market/ya_market/rest.py src/ya_market/rest.py
*** ya-py-aioclient-market/ya_market/rest.py	2020-06-29 01:12:21.747973508 +0200
--- src/ya_market/rest.py	2020-06-29 01:14:54.467658289 +0200
***************
*** 17,22 ****
--- 17,23 ----
  import ssl
  
  import aiohttp
+ from urllib.parse import urlencode
  
  from ya_market.exceptions import ApiException, ApiValueError
  
Only in ya-py-aioclient-market/ya_market: rest.py.orig
