Only in src/ya_activity/api: __pycache__
diff -r -c target/ya_activity/api/requestor_control_api.py src/ya_activity/api/requestor_control_api.py
*** target/ya_activity/api/requestor_control_api.py	2020-12-05 15:48:50.317000000 +0100
--- src/ya_activity/api/requestor_control_api.py	2020-12-05 15:51:50.779920615 +0100
***************
*** 610,616 ****
              ]
          )

!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 610,616 ----
              ]
          )

!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
diff -r -c target/ya_activity/models/command_output_bin.py src/ya_activity/models/command_output_bin.py
*** target/ya_activity/models/command_output_bin.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/command_output_bin.py	2020-12-05 15:52:33.347587457 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/command_output_str.py src/ya_activity/models/command_output_str.py
*** target/ya_activity/models/command_output_str.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/command_output_str.py	2020-12-05 15:52:33.487586366 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/create_activity_request.py src/ya_activity/models/create_activity_request.py
*** target/ya_activity/models/create_activity_request.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/create_activity_request.py	2020-12-05 15:52:33.207588550 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 104,110 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 102,108 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/create_activity_result.py src/ya_activity/models/create_activity_result.py
*** target/ya_activity/models/create_activity_result.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/create_activity_result.py	2020-12-05 15:52:33.367587301 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 104,110 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 102,108 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/credentials.py src/ya_activity/models/credentials.py
*** target/ya_activity/models/credentials.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/credentials.py	2020-12-05 15:52:33.411586959 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 78,84 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 76,82 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/deploy_command.py src/ya_activity/models/deploy_command.py
*** target/ya_activity/models/deploy_command.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/deploy_command.py	2020-12-05 15:52:33.419586896 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/file_set.py src/ya_activity/models/file_set.py
*** target/ya_activity/models/file_set.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/file_set.py	2020-12-05 15:52:33.351587427 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 129,135 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 127,133 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
Only in src/ya_activity/models: __pycache__
diff -r -c target/ya_activity/models/run_command_body.py src/ya_activity/models/run_command_body.py
*** target/ya_activity/models/run_command_body.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/run_command_body.py	2020-12-05 15:52:33.463586554 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 103,109 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 101,107 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/run_command.py src/ya_activity/models/run_command.py
*** target/ya_activity/models/run_command.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/run_command.py	2020-12-05 15:52:33.151588985 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/runtime_event_kind_finished_body.py src/ya_activity/models/runtime_event_kind_finished_body.py
*** target/ya_activity/models/runtime_event_kind_finished_body.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/runtime_event_kind_finished_body.py	2020-12-05 15:52:33.183588736 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 103,109 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 101,107 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/runtime_event_kind_finished.py src/ya_activity/models/runtime_event_kind_finished.py
*** target/ya_activity/models/runtime_event_kind_finished.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/runtime_event_kind_finished.py	2020-12-05 15:52:33.243588269 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/runtime_event_kind_started.py src/ya_activity/models/runtime_event_kind_started.py
*** target/ya_activity/models/runtime_event_kind_started.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/runtime_event_kind_started.py	2020-12-05 15:52:33.223588425 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 79,85 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 77,83 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/runtime_event_kind_std_err.py src/ya_activity/models/runtime_event_kind_std_err.py
*** target/ya_activity/models/runtime_event_kind_std_err.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/runtime_event_kind_std_err.py	2020-12-05 15:52:33.443586710 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/runtime_event_kind_std_out.py src/ya_activity/models/runtime_event_kind_std_out.py
*** target/ya_activity/models/runtime_event_kind_std_out.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/runtime_event_kind_std_out.py	2020-12-05 15:52:33.431586804 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/runtime_event.py src/ya_activity/models/runtime_event.py
*** target/ya_activity/models/runtime_event.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/runtime_event.py	2020-12-05 15:52:33.399587052 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 155,161 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 153,159 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/sgx_credentials.py src/ya_activity/models/sgx_credentials.py
*** target/ya_activity/models/sgx_credentials.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/sgx_credentials.py	2020-12-05 15:52:33.163588892 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 186,192 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 184,190 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/start_command_body.py src/ya_activity/models/start_command_body.py
*** target/ya_activity/models/start_command_body.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/start_command_body.py	2020-12-05 15:52:33.091589453 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/start_command.py src/ya_activity/models/start_command.py
*** target/ya_activity/models/start_command.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/start_command.py	2020-12-05 15:52:33.107589329 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/transfer_command_body.py src/ya_activity/models/transfer_command_body.py
*** target/ya_activity/models/transfer_command_body.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/transfer_command_body.py	2020-12-05 15:52:33.495586304 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 181,187 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 179,185 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/transfer_command.py src/ya_activity/models/transfer_command.py
*** target/ya_activity/models/transfer_command.py	2020-12-05 15:48:50.261000000 +0100
--- src/ya_activity/models/transfer_command.py	2020-12-05 15:52:33.147589017 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401

- import six
-
  from ya_activity.configuration import Configuration


--- 13,18 ----
***************
*** 77,83 ****
          """Returns the model properties as a dict"""
          result = {}

!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 75,81 ----
          """Returns the model properties as a dict"""
          result = {}

!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/test/test_deploy_command_all_of.py src/ya_activity/test/test_deploy_command_all_of.py
*** target/ya_activity/test/test_deploy_command_all_of.py	2020-12-07 15:08:48.433000000 +0100
--- src/ya_activity/test/test_deploy_command_all_of.py	2020-12-07 15:15:31.709301349 +0100
***************
*** 36,46 ****
          # model = ya_activity.models.deploy_command_all_of.DeployCommandAllOf()  # noqa: E501
          if include_optional :
              return DeployCommandAllOf(
!                 deploy = None
              )
          else :
              return DeployCommandAllOf(
!                 deploy = None,
          )

      def testDeployCommandAllOf(self):
--- 36,46 ----
          # model = ya_activity.models.deploy_command_all_of.DeployCommandAllOf()  # noqa: E501
          if include_optional :
              return DeployCommandAllOf(
!                 deploy = dict()
              )
          else :
              return DeployCommandAllOf(
!                 deploy = dict(),
          )

      def testDeployCommandAllOf(self):
diff -r -c target/ya_activity/test/test_deploy_command.py src/ya_activity/test/test_deploy_command.py
*** target/ya_activity/test/test_deploy_command.py	2020-12-07 15:08:48.433000000 +0100
--- src/ya_activity/test/test_deploy_command.py	2020-12-07 15:13:52.330411109 +0100
***************
*** 36,46 ****
          # model = ya_activity.models.deploy_command.DeployCommand()  # noqa: E501
          if include_optional :
              return DeployCommand(
!                 deploy = None
              )
          else :
              return DeployCommand(
!                 deploy = None,
          )

      def testDeployCommand(self):
--- 36,46 ----
          # model = ya_activity.models.deploy_command.DeployCommand()  # noqa: E501
          if include_optional :
              return DeployCommand(
!                 deploy = dict()
              )
          else :
              return DeployCommand(
!                 deploy = dict(),
          )

      def testDeployCommand(self):
diff -r -c target/ya_activity/test/test_runtime_event_kind_started_all_of.py src/ya_activity/test/test_runtime_event_kind_started_all_of.py
*** target/ya_activity/test/test_runtime_event_kind_started_all_of.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_runtime_event_kind_started_all_of.py	2020-12-07 15:16:15.108816288 +0100
***************
*** 34,46 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_started_all_of.RuntimeEventKindStartedAllOf()  # noqa: E501
          if include_optional :
              return RuntimeEventKindStartedAllOf(
!                 started = None
              )
          else :
              return RuntimeEventKindStartedAllOf(
!                 started = None,
          )

      def testRuntimeEventKindStartedAllOf(self):
--- 34,47 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_started_all_of.RuntimeEventKindStartedAllOf()  # noqa: E501
+         command = ya_activity.models.DeployCommand(deploy=dict())
          if include_optional :
              return RuntimeEventKindStartedAllOf(
!                 started = command
              )
          else :
              return RuntimeEventKindStartedAllOf(
!                 started = command,
          )

      def testRuntimeEventKindStartedAllOf(self):
diff -r -c target/ya_activity/test/test_runtime_event_kind_started.py src/ya_activity/test/test_runtime_event_kind_started.py
*** target/ya_activity/test/test_runtime_event_kind_started.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_runtime_event_kind_started.py	2020-12-07 15:16:05.216926865 +0100
***************
*** 34,46 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_started.RuntimeEventKindStarted()  # noqa: E501
          if include_optional :
              return RuntimeEventKindStarted(
!                 started = None
              )
          else :
              return RuntimeEventKindStarted(
!                 started = None,
          )

      def testRuntimeEventKindStarted(self):
--- 34,47 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_started.RuntimeEventKindStarted()  # noqa: E501
+         command = ya_activity.models.DeployCommand(deploy=dict())
          if include_optional :
              return RuntimeEventKindStarted(
!                 started = command
              )
          else :
              return RuntimeEventKindStarted(
!                 started = command,
          )

      def testRuntimeEventKindStarted(self):
diff -r -c target/ya_activity/test/test_runtime_event_kind_std_err_all_of.py src/ya_activity/test/test_runtime_event_kind_std_err_all_of.py
*** target/ya_activity/test/test_runtime_event_kind_std_err_all_of.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_runtime_event_kind_std_err_all_of.py	2020-12-07 15:19:29.762638252 +0100
***************
*** 34,46 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_std_err_all_of.RuntimeEventKindStdErrAllOf()  # noqa: E501
          if include_optional :
              return RuntimeEventKindStdErrAllOf(
!                 stderr = None
              )
          else :
              return RuntimeEventKindStdErrAllOf(
!                 stderr = None,
          )

      def testRuntimeEventKindStdErrAllOf(self):
--- 34,47 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_std_err_all_of.RuntimeEventKindStdErrAllOf()  # noqa: E501
+         out = ya_activity.models.CommandOutputStr(str="0")
          if include_optional :
              return RuntimeEventKindStdErrAllOf(
!                 stderr = out
              )
          else :
              return RuntimeEventKindStdErrAllOf(
!                 stderr = out,
          )

      def testRuntimeEventKindStdErrAllOf(self):
diff -r -c target/ya_activity/test/test_runtime_event_kind_std_err.py src/ya_activity/test/test_runtime_event_kind_std_err.py
*** target/ya_activity/test/test_runtime_event_kind_std_err.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_runtime_event_kind_std_err.py	2020-12-07 15:16:34.792596219 +0100
***************
*** 34,46 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_std_err.RuntimeEventKindStdErr()  # noqa: E501
          if include_optional :
              return RuntimeEventKindStdErr(
!                 stderr = None
              )
          else :
              return RuntimeEventKindStdErr(
!                 stderr = None,
          )

      def testRuntimeEventKindStdErr(self):
--- 34,47 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_std_err.RuntimeEventKindStdErr()  # noqa: E501
+         out = ya_activity.models.CommandOutputStr(str="0")
          if include_optional :
              return RuntimeEventKindStdErr(
!                 stderr = out,
              )
          else :
              return RuntimeEventKindStdErr(
!                 stderr = out,
          )

      def testRuntimeEventKindStdErr(self):
diff -r -c target/ya_activity/test/test_runtime_event_kind_std_out_all_of.py src/ya_activity/test/test_runtime_event_kind_std_out_all_of.py
*** target/ya_activity/test/test_runtime_event_kind_std_out_all_of.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_runtime_event_kind_std_out_all_of.py	2020-12-07 15:19:29.742638477 +0100
***************
*** 34,46 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_std_out_all_of.RuntimeEventKindStdOutAllOf()  # noqa: E501
          if include_optional :
              return RuntimeEventKindStdOutAllOf(
!                 stdout = None
              )
          else :
              return RuntimeEventKindStdOutAllOf(
!                 stdout = None,
          )

      def testRuntimeEventKindStdOutAllOf(self):
--- 34,47 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_std_out_all_of.RuntimeEventKindStdOutAllOf()  # noqa: E501
+         out = ya_activity.models.CommandOutputStr(str="0")
          if include_optional :
              return RuntimeEventKindStdOutAllOf(
!                 stdout = out
              )
          else :
              return RuntimeEventKindStdOutAllOf(
!                 stdout = out,
          )

      def testRuntimeEventKindStdOutAllOf(self):
diff -r -c target/ya_activity/test/test_runtime_event_kind_std_out.py src/ya_activity/test/test_runtime_event_kind_std_out.py
*** target/ya_activity/test/test_runtime_event_kind_std_out.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_runtime_event_kind_std_out.py	2020-12-07 15:19:29.754638341 +0100
***************
*** 34,46 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_std_out.RuntimeEventKindStdOut()  # noqa: E501
          if include_optional :
              return RuntimeEventKindStdOut(
!                 stdout = None
              )
          else :
              return RuntimeEventKindStdOut(
!                 stdout = None,
          )

      def testRuntimeEventKindStdOut(self):
--- 34,47 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event_kind_std_out.RuntimeEventKindStdOut()  # noqa: E501
+         out = ya_activity.models.CommandOutputStr(str="0")
          if include_optional :
              return RuntimeEventKindStdOut(
!                 stdout = out
              )
          else :
              return RuntimeEventKindStdOut(
!                 stdout = out,
          )

      def testRuntimeEventKindStdOut(self):
diff -r -c target/ya_activity/test/test_runtime_event.py src/ya_activity/test/test_runtime_event.py
*** target/ya_activity/test/test_runtime_event.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_runtime_event.py	2020-12-07 15:15:31.701301439 +0100
***************
*** 34,52 ****
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event.RuntimeEvent()  # noqa: E501
          if include_optional :
              return RuntimeEvent(
!                 batch_id = '0', 
!                 index = 56, 
!                 timestamp = '0', 
!                 kind = None
              )
          else :
              return RuntimeEvent(
                  batch_id = '0',
                  index = 56,
                  timestamp = '0',
!                 kind = None,
          )

      def testRuntimeEvent(self):
--- 34,53 ----
              params are included, when True both required and
              optional params are included """
          # model = ya_activity.models.runtime_event.RuntimeEvent()  # noqa: E501
+         started = ya_activity.models.RuntimeEventKindStarted(started=dict())
          if include_optional :
              return RuntimeEvent(
!                 batch_id = '0',
!                 index = 56,
!                 timestamp = '0',
!                 kind = started
              )
          else :
              return RuntimeEvent(
                  batch_id = '0',
                  index = 56,
                  timestamp = '0',
!                 kind = started,
          )

      def testRuntimeEvent(self):
diff -r -c target/ya_activity/test/test_sign_command_all_of.py src/ya_activity/test/test_sign_command_all_of.py
*** target/ya_activity/test/test_sign_command_all_of.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_sign_command_all_of.py	2020-12-07 16:26:26.128050160 +0100
***************
*** 36,46 ****
          # model = ya_activity.models.sign_command_all_of.SignCommandAllOf()  # noqa: E501
          if include_optional :
              return SignCommandAllOf(
!                 sign = None
              )
          else :
              return SignCommandAllOf(
!                 sign = None,
          )

      def testSignCommandAllOf(self):
--- 36,46 ----
          # model = ya_activity.models.sign_command_all_of.SignCommandAllOf()  # noqa: E501
          if include_optional :
              return SignCommandAllOf(
!                 sign = dict()
              )
          else :
              return SignCommandAllOf(
!                 sign = dict(),
          )

      def testSignCommandAllOf(self):
diff -r -c target/ya_activity/test/test_sign_command.py src/ya_activity/test/test_sign_command.py
*** target/ya_activity/test/test_sign_command.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_sign_command.py	2020-12-07 16:26:13.352212870 +0100
***************
*** 36,46 ****
          # model = ya_activity.models.sign_command.SignCommand()  # noqa: E501
          if include_optional :
              return SignCommand(
!                 sign = None
              )
          else :
              return SignCommand(
!                 sign = None,
          )

      def testSignCommand(self):
--- 36,46 ----
          # model = ya_activity.models.sign_command.SignCommand()  # noqa: E501
          if include_optional :
              return SignCommand(
!                 sign = dict()
              )
          else :
              return SignCommand(
!                 sign = dict(),
          )

      def testSignCommand(self):
diff -r -c target/ya_activity/test/test_terminate_command_all_of.py src/ya_activity/test/test_terminate_command_all_of.py
*** target/ya_activity/test/test_terminate_command_all_of.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_terminate_command_all_of.py	2020-12-07 16:26:34.139948145 +0100
***************
*** 36,46 ****
          # model = ya_activity.models.terminate_command_all_of.TerminateCommandAllOf()  # noqa: E501
          if include_optional :
              return TerminateCommandAllOf(
!                 terminate = None
              )
          else :
              return TerminateCommandAllOf(
!                 terminate = None,
          )

      def testTerminateCommandAllOf(self):
--- 36,46 ----
          # model = ya_activity.models.terminate_command_all_of.TerminateCommandAllOf()  # noqa: E501
          if include_optional :
              return TerminateCommandAllOf(
!                 terminate = dict()
              )
          else :
              return TerminateCommandAllOf(
!                 terminate = dict(),
          )

      def testTerminateCommandAllOf(self):
diff -r -c target/ya_activity/test/test_terminate_command.py src/ya_activity/test/test_terminate_command.py
*** target/ya_activity/test/test_terminate_command.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_terminate_command.py	2020-12-07 16:26:34.151947992 +0100
***************
*** 36,46 ****
          # model = ya_activity.models.terminate_command.TerminateCommand()  # noqa: E501
          if include_optional :
              return TerminateCommand(
!                 terminate = None
              )
          else :
              return TerminateCommand(
!                 terminate = None,
          )

      def testTerminateCommand(self):
--- 36,46 ----
          # model = ya_activity.models.terminate_command.TerminateCommand()  # noqa: E501
          if include_optional :
              return TerminateCommand(
!                 terminate = dict()
              )
          else :
              return TerminateCommand(
!                 terminate = dict(),
          )

      def testTerminateCommand(self):
diff -r -c target/ya_activity/test/test_transfer_command_all_of.py src/ya_activity/test/test_transfer_command_all_of.py
*** target/ya_activity/test/test_transfer_command_all_of.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_transfer_command_all_of.py	2020-12-07 15:10:05.988932346 +0100
***************
*** 39,40 ****
                  transfer = ya_activity.models.transfer_command_body.TransferCommandBody(
!                     from = '0', 
--- 39,40 ----
                  transfer = ya_activity.models.transfer_command_body.TransferCommandBody(
!                     _from = '0',
***************
*** 57,58 ****
                  transfer = ya_activity.models.transfer_command_body.TransferCommandBody(
!                     from = '0', 
--- 57,58 ----
                  transfer = ya_activity.models.transfer_command_body.TransferCommandBody(
!                     _from = '0',
diff -r -c target/ya_activity/test/test_transfer_command.py src/ya_activity/test/test_transfer_command.py
*** target/ya_activity/test/test_transfer_command.py	2020-12-07 15:08:48.437000000 +0100
--- src/ya_activity/test/test_transfer_command.py	2020-12-07 15:10:05.996932257 +0100
***************
*** 39,40 ****
                  transfer = ya_activity.models.transfer_command_body.TransferCommandBody(
!                     from = '0', 
--- 39,40 ----
                  transfer = ya_activity.models.transfer_command_body.TransferCommandBody(
!                     _from = '0',
***************
*** 57,58 ****
                  transfer = ya_activity.models.transfer_command_body.TransferCommandBody(
!                     from = '0', 
--- 57,58 ----
                  transfer = ya_activity.models.transfer_command_body.TransferCommandBody(
!                     _from = '0',
