diff -r -c target/ya_activity/api/requestor_control_api.py src/ya_activity/api/requestor_control_api.py
*** target/ya_activity/api/requestor_control_api.py	2021-03-22 17:09:16.944000000 +0100
--- src/ya_activity/api/requestor_control_api.py	2021-03-22 17:14:45.367197913 +0100
***************
*** 362,368 ****
              ]
          )
  
!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 362,368 ----
              ]
          )
  
!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
