diff -r -c target/ya_activity/models/deploy_command_body.py src/ya_activity/models/deploy_command_body.py
*** target/ya_activity/models/deploy_command_body.py	2021-04-20 15:41:15.389000000 +0200
--- src/ya_activity/models/deploy_command_body.py	2021-04-20 15:42:32.976915765 +0200
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401
  
- import six
- 
  from ya_activity.configuration import Configuration
  
  
--- 13,18 ----
***************
*** 129,135 ****
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 127,133 ----
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_activity/models/deploy_network.py src/ya_activity/models/deploy_network.py
*** target/ya_activity/models/deploy_network.py	2021-04-20 15:41:15.389000000 +0200
--- src/ya_activity/models/deploy_network.py	2021-04-20 15:43:42.428016011 +0200
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401
  
- import six
- 
  from ya_activity.configuration import Configuration
  
  
--- 13,18 ----
***************
*** 132,138 ****
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 130,136 ----
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
