diff -r -c target/ya_payment/rest.py src/ya_payment/rest.py
*** target/ya_payment/rest.py	2020-10-02 09:21:59.428267699 +0200
--- src/ya_payment/rest.py	2020-10-02 09:27:26.473942367 +0200
***************
*** 165,170 ****
--- 165,172 ----
                           declared content type."""
                  raise ApiException(status=0, reason=msg)
  
+         logger.debug("request: %s", args)
+ 
          r = await self.pool_manager.request(**args)
          if _preload_content:
  
