diff -r -c target/ya_payment/api/requestor_api.py src/ya_payment/api/requestor_api.py
*** target/ya_payment/api/requestor_api.py	2021-03-22 17:09:16.904000000 +0100
--- src/ya_payment/api/requestor_api.py	2021-03-22 17:13:38.964174530 +0100
***************
*** 2296,2302 ****
              ]
          )
  
!         for key, val in six.iteritems(local_var_params['kwargs']):
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
--- 2296,2302 ----
              ]
          )
  
!         for key, val in local_var_params['kwargs'].items():
              if key not in all_params:
                  raise ApiTypeError(
                      "Got an unexpected keyword argument '%s'"
