diff -r -c target/ya_payment/models/debit_note_payment_ok_event.py src/ya_payment/models/debit_note_payment_ok_event.py
*** target/ya_payment/models/debit_note_payment_ok_event.py	2023-11-21 12:49:47.171339218 +0100
--- src/ya_payment/models/debit_note_payment_ok_event.py	2023-11-21 12:57:49.773697452 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401
  
- import six
- 
  from ya_payment.configuration import Configuration
  
  
--- 13,18 ----
***************
*** 33,54 ****
                              and the value is json key in definition.
      """
      openapi_types = {
          'debit_note_id': 'str'
      }
  
      attribute_map = {
          'debit_note_id': 'debitNoteId'
      }
  
!     def __init__(self, debit_note_id=None, local_vars_configuration=None):  # noqa: E501
          """DebitNotePaymentOkEvent - a model defined in OpenAPI"""  # noqa: E501
          if local_vars_configuration is None:
              local_vars_configuration = Configuration()
          self.local_vars_configuration = local_vars_configuration
  
          self._debit_note_id = None
          self.discriminator = None
  
          if debit_note_id is not None:
              self.debit_note_id = debit_note_id
  
--- 31,57 ----
                              and the value is json key in definition.
      """
      openapi_types = {
+         'event_date': 'datetime',
          'debit_note_id': 'str'
      }
  
      attribute_map = {
+         'event_date': 'eventDate',
          'debit_note_id': 'debitNoteId'
      }
  
!     def __init__(self, debit_note_id=None, local_vars_configuration=None, event_date=None):  # noqa: E501
          """DebitNotePaymentOkEvent - a model defined in OpenAPI"""  # noqa: E501
          if local_vars_configuration is None:
              local_vars_configuration = Configuration()
          self.local_vars_configuration = local_vars_configuration
  
          self._debit_note_id = None
+         self._event_date = None
          self.discriminator = None
  
+         if event_date is not None:
+             self.event_date = event_date
          if debit_note_id is not None:
              self.debit_note_id = debit_note_id
  
***************
*** 73,83 ****
  
          self._debit_note_id = debit_note_id
  
      def to_dict(self):
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 76,94 ----
  
          self._debit_note_id = debit_note_id
  
+     @property
+     def event_date(self):
+         return self._event_date
+ 
+     @event_date.setter
+     def event_date(self, event_date):
+         self._event_date = event_date
+ 
      def to_dict(self):
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_payment/models/debit_note_payment_status_event.py src/ya_payment/models/debit_note_payment_status_event.py
*** target/ya_payment/models/debit_note_payment_status_event.py	2023-11-21 12:49:47.171339218 +0100
--- src/ya_payment/models/debit_note_payment_status_event.py	2023-11-21 12:57:59.029776883 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401
  
- import six
- 
  from ya_payment.configuration import Configuration
  
  
--- 13,18 ----
***************
*** 33,57 ****
                              and the value is json key in definition.
      """
      openapi_types = {
          'debit_note_id': 'str',
          '_property': 'DriverStatusProperty'
      }
  
      attribute_map = {
          'debit_note_id': 'debitNoteId',
          '_property': 'property'
      }
  
!     def __init__(self, debit_note_id=None, _property=None, local_vars_configuration=None):  # noqa: E501
          """DebitNotePaymentStatusEvent - a model defined in OpenAPI"""  # noqa: E501
          if local_vars_configuration is None:
              local_vars_configuration = Configuration()
          self.local_vars_configuration = local_vars_configuration
  
          self._debit_note_id = None
          self.__property = None
          self.discriminator = None
  
          if debit_note_id is not None:
              self.debit_note_id = debit_note_id
          if _property is not None:
--- 31,60 ----
                              and the value is json key in definition.
      """
      openapi_types = {
+         'event_date': 'datetime',
          'debit_note_id': 'str',
          '_property': 'DriverStatusProperty'
      }
  
      attribute_map = {
+         'event_date': 'eventDate',
          'debit_note_id': 'debitNoteId',
          '_property': 'property'
      }
  
!     def __init__(self, debit_note_id=None, _property=None, local_vars_configuration=None, event_date=None):  # noqa: E501
          """DebitNotePaymentStatusEvent - a model defined in OpenAPI"""  # noqa: E501
          if local_vars_configuration is None:
              local_vars_configuration = Configuration()
          self.local_vars_configuration = local_vars_configuration
  
          self._debit_note_id = None
+         self._event_date = None
          self.__property = None
          self.discriminator = None
  
+         if event_date is not None:
+             self.event_date = event_date
          if debit_note_id is not None:
              self.debit_note_id = debit_note_id
          if _property is not None:
***************
*** 79,84 ****
--- 82,95 ----
          self._debit_note_id = debit_note_id
  
      @property
+     def event_date(self):
+         return self._event_date
+ 
+     @event_date.setter
+     def event_date(self, event_date):
+         self._event_date = event_date
+ 
+     @property
      def _property(self):
          """Gets the _property of this DebitNotePaymentStatusEvent.  # noqa: E501
  
***************
*** 103,109 ****
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 114,120 ----
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_payment/models/invoice_payment_ok_event.py src/ya_payment/models/invoice_payment_ok_event.py
*** target/ya_payment/models/invoice_payment_ok_event.py	2023-11-21 12:49:47.171339218 +0100
--- src/ya_payment/models/invoice_payment_ok_event.py	2023-11-21 13:26:02.637910634 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401
  
- import six
- 
  from ya_payment.configuration import Configuration
  
  
--- 13,18 ----
***************
*** 33,54 ****
                              and the value is json key in definition.
      """
      openapi_types = {
          'invoice_id': 'str'
      }
  
      attribute_map = {
          'invoice_id': 'invoiceId'
      }
  
!     def __init__(self, invoice_id=None, local_vars_configuration=None):  # noqa: E501
          """InvoicePaymentOkEvent - a model defined in OpenAPI"""  # noqa: E501
          if local_vars_configuration is None:
              local_vars_configuration = Configuration()
          self.local_vars_configuration = local_vars_configuration
  
          self._invoice_id = None
          self.discriminator = None
  
          if invoice_id is not None:
              self.invoice_id = invoice_id
  
--- 31,57 ----
                              and the value is json key in definition.
      """
      openapi_types = {
+         'event_date': 'datetime',
          'invoice_id': 'str'
      }
  
      attribute_map = {
+         'event_date': 'eventDate',
          'invoice_id': 'invoiceId'
      }
  
!     def __init__(self, invoice_id=None, local_vars_configuration=None, event_date=None):  # noqa: E501
          """InvoicePaymentOkEvent - a model defined in OpenAPI"""  # noqa: E501
          if local_vars_configuration is None:
              local_vars_configuration = Configuration()
          self.local_vars_configuration = local_vars_configuration
  
          self._invoice_id = None
+         self._event_date = None
          self.discriminator = None
  
+         if event_date is not None:
+             self.event_date = event_date
          if invoice_id is not None:
              self.invoice_id = invoice_id
  
***************
*** 73,83 ****
  
          self._invoice_id = invoice_id
  
      def to_dict(self):
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 76,94 ----
  
          self._invoice_id = invoice_id
  
+     @property
+     def event_date(self):
+         return self._event_date
+ 
+     @event_date.setter
+     def event_date(self, event_date):
+         self._event_date = event_date
+ 
      def to_dict(self):
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
diff -r -c target/ya_payment/models/invoice_payment_status_event.py src/ya_payment/models/invoice_payment_status_event.py
*** target/ya_payment/models/invoice_payment_status_event.py	2023-11-21 12:49:47.171339218 +0100
--- src/ya_payment/models/invoice_payment_status_event.py	2023-11-21 13:26:10.681795886 +0100
***************
*** 13,20 ****
  import pprint
  import re  # noqa: F401
  
- import six
- 
  from ya_payment.configuration import Configuration
  
  
--- 13,18 ----
***************
*** 33,57 ****
                              and the value is json key in definition.
      """
      openapi_types = {
          'invoice_id': 'str',
          '_property': 'DriverStatusProperty'
      }
  
      attribute_map = {
          'invoice_id': 'invoiceId',
          '_property': 'property'
      }
  
!     def __init__(self, invoice_id=None, _property=None, local_vars_configuration=None):  # noqa: E501
          """InvoicePaymentStatusEvent - a model defined in OpenAPI"""  # noqa: E501
          if local_vars_configuration is None:
              local_vars_configuration = Configuration()
          self.local_vars_configuration = local_vars_configuration
  
          self._invoice_id = None
          self.__property = None
          self.discriminator = None
  
          if invoice_id is not None:
              self.invoice_id = invoice_id
          if _property is not None:
--- 31,60 ----
                              and the value is json key in definition.
      """
      openapi_types = {
+         'event_date': 'datetime',
          'invoice_id': 'str',
          '_property': 'DriverStatusProperty'
      }
  
      attribute_map = {
+         'event_date': 'eventDate',
          'invoice_id': 'invoiceId',
          '_property': 'property'
      }
  
!     def __init__(self, invoice_id=None, _property=None, local_vars_configuration=None, event_date=None):  # noqa: E501
          """InvoicePaymentStatusEvent - a model defined in OpenAPI"""  # noqa: E501
          if local_vars_configuration is None:
              local_vars_configuration = Configuration()
          self.local_vars_configuration = local_vars_configuration
  
          self._invoice_id = None
+         self._event_date = None
          self.__property = None
          self.discriminator = None
  
+         if event_date is not None:
+             self.event_date = event_date
          if invoice_id is not None:
              self.invoice_id = invoice_id
          if _property is not None:
***************
*** 79,84 ****
--- 82,95 ----
          self._invoice_id = invoice_id
  
      @property
+     def event_date(self):
+         return self._event_date
+ 
+     @event_date.setter
+     def event_date(self, event_date):
+         self._event_date = event_date
+ 
+     @property
      def _property(self):
          """Gets the _property of this InvoicePaymentStatusEvent.  # noqa: E501
  
***************
*** 103,109 ****
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr, _ in six.iteritems(self.openapi_types):
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
--- 114,120 ----
          """Returns the model properties as a dict"""
          result = {}
  
!         for attr in self.openapi_types:
              value = getattr(self, attr)
              if isinstance(value, list):
                  result[attr] = list(map(
